tokio-trace/tokio-trace-core/src/metadata.rs

```diff
@@ -154,6 +174,8 @@ impl<'a> Metadata<'a> {
                 names: field_names,
                 callsite: callsite::Identifier(callsite),
             },
+            // FIXME(csmoe): what the default callsite kind should be?
```

Comment:
We probably don't want to have a default callsite kind --- instead, I think any time a new `Metadata` is constructed, we should _require_ a kind be provided.